### ISSUE TITLE
TextField in tableview cell 

### ISSUE THREAD

**Issue by mirzadelic** [gwadhwani/MPGTextField#24]
I can't see suggestions when typing.. i have tableview cell and textfield in it.
Any solution for this?

Image:
![simulator screen shot nov 13 2015 00 43 21](https://cloud.githubusercontent.com/assets/1878794/11134781/c97676e8-899f-11e5-85f2-617dfaadd7a5.png)

**Comment by willsmillie** [comment]
I am also having this problem.

**Comment by pavankataria** [comment]
Yeah, what's annoying about this is that the cell constrains the touch events 
Damn, it's been one year already and still no solution, any news on this?
@mirzadelic what did you do to fix this issue?

**Comment by pavankataria** [comment]
@williamfs were you able to do anything to get this working?
